applications/pfem_2_application/custom_elements/qfluid_2d.h

```diff
@@ -0,0 +1,343 @@
+/*
+==============================================================================
```

Comment:
Just as a comment: this is the old license and it would be better to include the new header from [template](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/templates/headeronly_template#L1)